### DISCUSSION TITLE
Dashboard for Deye 5K Lp1

### DISCUSSION THREAD

**Discussion by concierge-at** [slipx06/sunsynk-power-flow-card#479] [Sunsynk/Deye]
Hello, thank you for your project, I would like to use your dasboard with your ESP code for the LP1 Deye 5K. 

I then added a new card and inserted the code from Dashboard.yaml under Edit Yaml, saved it and the field remains invisible. I also installed the extensions in HACS. I haven't installed loadsheding and solarxast because I won't be using them.

Can you please tell me where I made a mistake so that the views for statistics and performance flow are not displayed

Please excuse all the questions, but I'm still new to HA. In addition, English is not my native language so I do everything with a translator.
 I've tried it several times but always without success to install your beautifull dashboard 

thanks in advance and all the best

![HACS-install](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/84469df1-16e3-455c-9e84-12a5b4924559)
![HA-Site-yaml2](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/6948aad9-3506-4809-b7e3-9b7be1625ac1)
![HA-Site-yaml1](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/aec7b391-1f80-49e4-b725-83353a76d9f8)
![HA-Site-nothing](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/c4d6868d-0bce-4d21-be18-82f6bd66c031)
![HA-Site](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/ce76879d-4c30-433b-939b-e134c51309f3)
![5k2g03lp1-HA-ent-1b](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/42b98115-b169-4332-b72b-7867b3672cf9)

**Comment by slipx06** [comment]
Hello. The best approach would be to copy out the individual cards that you like and get them working. Then you can work on the layout and view. How are you reading information from your Inverter? Looks like there is a problem with your integration.

**Reply by concierge-at** [reply]
Hello and thank you for the quick response, 
I read the data with (https://github.com/slipx06/Sunsynk-Home-Assistant-Dash/blob/main/ESPHome-1P-Sunsynk-Deye.yaml) via the ESP Rs485 adapter. 

Edit: i use the dashboard code from here https://github.com/slipx06/Sunsynk-Home-Assistant-Dash/blob/main/Dashboard.yaml

Could it be because the adapter is not yet connected to the inverter? I wanted to get everything ready first so that I could drive there with the complete solution and hook it up to the inverter. But as you can see in the pictures, the entities are in the HA. Apparently I'm just too stupid for a dashboard. I don't even get an error message that something is missing. Then of course it is difficult to find the error. I would like the dashboard as it is, but (the problem is getting it to work)

I tried again and again and the window stays black (or white) depending on the view.

I'll ask again in a German forum, maybe someone can help me, otherwise it fails, unfortunately. It's probably just a step that I've forgotten, but I no longer know how to control myself.

I thank you in any case, but without programming knowledge I can't get any further 🤦‍♀️🤷‍♀️

**Reply by concierge-at** [reply]
> Hallo. Am besten kopierst du die einzelnen Karten, die dir gefallen, und bringst sie zum Laufen. Dann kannst du am Layout und der Ansicht arbeiten. Wie liest du die Informationen aus deinem Inverter? Es sieht so aus, als ob es ein Problem mit deiner Integration gibt.

Hello and good morning, thank you i have try to separate the code. I have the sunsync card  running and in an other tab the most of apex Charts.
But when i put them on the same card, the charts and card will not be show. When i press f5 on the keyboard for a second all will showen end then all will hide.
Maybe there is a problem in the install of homeassistant?

I have done reinstall all new on a new hard disk every time the same i am near to get crazy.🤔😭

**Reply by slipx06** [reply]
@concierge-at  I made this a while back to try explain how the layout card works with the grid area. Maybe it will help

https://photos.app.goo.gl/unhJubM2r4BkjW3A8

Also make sure you have enough resources to display the dashboard i.e. CPU etc,

**Reply by concierge-at** [reply]
Thank you, Thank you,

YOU mean the performance of PI4 with 4Gb Ram is to less. I will try to install on an x64 4core Cpu and give feedback. PI is ok But not for all

**Reply by slipx06** [reply]
Depends on what other addons etc you are running

**Reply by concierge-at** [reply]
Only one Chrone Tab for SOC  to Shelly for my Water heater :) i Just play with the tab and on the weekend wenn i drive to visit the Inverter i will test it with live data.
Thank you for your inspiration to never give up :) 
![lay-2](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/987049cf-a603-4a25-992a-3d5069457f89)
![lay-1](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/dd23b8ad-b9ad-450d-9d2e-fba30df96fea)

**Reply by concierge-at** [reply]
> Hallo. Am besten kopierst du die einzelnen Karten, die dir gefallen, und bringst sie zum Laufen. Dann kannst du am Layout und der Ansicht arbeiten. Wie liest du die Informationen aus deinem Inverter? Es sieht so aus, als ob es ein Problem mit deiner Integration gibt.

Thank you fpr your inspiration, now it ist wprking, my big problem wars to get data fpm the Inverter. I try 4 Methods and no one has success.
But now since one week the littl gizmos are jumping.
i only have problems with the size but better wear glasses :) 
i have done a lot of testing and try and error, not it runs but i dont know what i do here ^^
Thanky a lot for the Nice card
![dashboard-](https://github.com/slipx06/sunsynk-power-flow-card/assets/135752342/f02e3154-e467-483c-b2c6-52ce61bc0a06)

**Comment by concierge-at** [comment]
Hello again 👋, slipx06
I have a question, I mean it's more of a visual question,
You can display the battery charge in 40.5A and 2.5kw and the discharge in -40.5 and -2.5kw. I tried to activate the invert switch in the configuration but then it was completely twisted. I chose the color green as the charge and the color red as the discharge, which also works very well :) thanks in advance. If it's not possible, that's no problem

**Reply by slipx06** [reply]
Hi. Have you set `show_absolute: false` under the battery section. 

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/ce918404-1343-4d67-8c31-d870cae85064)

**Reply by concierge-at** [reply]
> Hallo. Haben Sie es `show_absolute: false`im Abschnitt „Batterie“ eingestellt?
> 

Thank you, thank you again, now i have set the switch. I think i must be blindet by so many options🫣 . This is realy a great work. 
Now i spend time at the chart Boards, may be i must come again to ask you again 

Thank you for your work

**Comment by concierge-at** [comment]
Thank you thank you thank you for your great work. I have now also got your Deye LP1-RS485-ESP2HA working. I spent the weekend converting all entities. I have a question about a value later, but I first have to install the sensor into the installation. I'll be back :) thank you for your help and patience
